### DISCUSSION TITLE
admin-points-4326

### DISCUSSION THREAD

**Discussion by Qctsu** [shortbread-tiles/shortbread-docs#66] [General]
Hey everyone,

It seems there's an issue with downloading admin-points-4326. 
Could you please take a look at it? Or perhaps provide another link? :)

**Comment by joto** [comment]
Can you tell us what admin-points-4326 is supposed to be and where you are trying to download it from?

**Comment by Nakaner** [comment]
See https://github.com/shortbread-tiles/shortbread-tilemaker/issues/34
